pkg/apis/registry/v1alpha3/types.go

```diff
@@ -66,6 +69,19 @@ type GarbageCollection struct {
 	TTL metav1.Duration `json:"ttl"`
 }
 
+// Proxy contains settings for a proxy used in the registry cache.
+type Proxy struct {
+	// HTTPProxy is used as HTTP_PROXY env in the StatefulSet.
```

Comment:
As shared above, the doc string reveals how it is implemented, not what the field does/represents.

Reply:
fixed